features/fields-generator/fields_mapping_generator.py

```diff
@@ -0,0 +1,218 @@
+
+    if "$ref" in propertyDef:
+        writeComponent (path, propertyDef.get("$ref"))
+
+    if "properties" in propertyDef:
+        properties = sorted(propertyDef.get("properties"))
+        for subProperty, subPropertyDef in properties:
+            writeProperty(path, subProperty, subPropertyDef)
+
+            if subProperty=="type":
+                print ("type")
+
+    if propertyDef.get('type') == "array":
+        if "$ref" in propertyDef.get('items'):
+            writeComponent (path, propertyDef.get("items").get("$ref"))
+
+
+def inOnderzoek(path):
+    if path.find("inOnderzoek")>-1:
+        return path
+
+    pl = path.split(".")
+    pl.insert(-1, "inOnderzoek")
+    return ".".join(pl)
+
+
+def unique(path):
+    if debug==True:
+        print(path.get("shortPath"))
+    
+    if path.get("shortPath")==path.get("fullPath"):
+        return True
+
+    if path.get("fullPath")==inOnderzoek(path.get("fullPath")):
+        if len(list(filter(lambda p: p.get("shortPath")==path.get("shortPath"), unfilteredFieldPaths)))==1:
+            return True
+    else:
+        if len(list(filter(lambda p: p.get("shortPath")==path.get("shortPath") and p.get("fullPath")!=inOnderzoek(path.get("fullPath")), unfilteredFieldPaths)))==1:
+            return True
+
+    return False
+
+    
+def removeDateField(path):
+    if "(D)" in path["fullPath"]:
+        path["shortPath"] = path.get("shortPath").replace("(D)", "")
+
+    if "(D)" in path["fullPath"]:
+        path["fullPath"] = path.get("fullPath").split("(D)")[0]
+
+    return path
+
+
+# lees command line arguments
+debug = False
+info = False
+settingsFileName = ""
+
+for argument in sys.argv:
+    if (argument[:8]=="--config"):
+        settingsFileName = argument[8:]
+    if (argument=="--debug" or argument=="-d"):
+        debug = True
+        info = True
+    elif (argument=="--info" or argument=="-i"):
+        info = True
+    elif (argument=="--help" or argument=="-h"):
+        print ('Command line arguments:')
+        print ('--config         bestand van projectspecifieke configuratie json bestand')
+        print (' --info    -i    toon voortgangs info')
+        print (' --debug   -d    toon debug info')
+        print (' --help    -h    toon help op command line arguments')
+        sys.exit()
+
+# lees configuratiebestand
+if settingsFileName!="":
+    with open(settingsFileName) as settingsFile:
+        SETTINGS = json.load(settingsFile)
+else:
+    print ("Geef het gewenste config bestand op met parameter --config.")
+    sys.exit()
+
+if (not os.path.isdir(SETTINGS.get("projectFolder"))):
+    print ("Geef een correcte projectFolder in het config bestand.")
+    sys.exit()
+
+if (not os.path.isfile(SETTINGS.get("projectFolder") + SETTINGS.get("sourceYaml"))):
+    print ("Geef een correcte locatie op voor de API specificaties in sourceYaml.")
+    sys.exit()
+
+if (not os.path.isdir(SETTINGS.get("projectFolder") + SETTINGS.get("sheetFolder"))):
```

Comment:
sheetFolder komt niet voor als setting in brp_persoon.json

Reply:
foutje in de code, moet zijn fieldslistFolder